docs/index.rst

```diff
@@ -72,20 +72,23 @@ Step 2: visualize tracked data
 
 ------------
 
-**Guides**
+**Content**
 
 .. toctree::
   :maxdepth: 2
 
-  Guides <guides/index>
+  Documentation Home <self>
 
+.. toctree::
+  :maxdepth: 2
 
-**API references**
+  Guides <guides/index>
 
 .. toctree::
   :maxdepth: 2
+  :caption: References
```

Comment:
I don't think this is necessary. Your left sidebar is small enough to not need a caption to have distinct sections. I recommend consolidating all of the `.. toctree::`s into a single thing for simplicity since you're using the same `:maxdepth: 2` for them all.